common-arcana.lic

```diff
@@ -658,4 +658,89 @@ module DRCA
       cast_spells(spells, settings, settings.waggle_force_cambrinth)
     end
   end
+
+  def add_avtalia_flags(settings)
```

Comment:
❓ Why put these helpers in `DRCA` (which is already pretty big) if they're only used by one script? Do you plan to use them in other scripts later?

Reply:
You're right.  After building a few things with this, the flag add, remove, and process methods will only be used in avtalia.lic and nothing else should call them.  The other three will need to stay in DRCA to be called by others as needed.  I'll change them around.